xla/backends/gpu/collectives/nccl_collectives.cc

```diff
@@ -114,6 +115,15 @@ static ncclConfig_t AsNcclConfig(const GpuCollectives::Config& config) {
   if (config.max_nchannels > 0) {
     VLOG(1) << "Maximum number of channels is set to: " << comm_config.maxCTAs;
     comm_config.maxCTAs = config.max_nchannels;
+  } else if (stream_executor->GetDeviceDescription()
+                 .cuda_compute_capability()
+                 .IsBlackwell() &&
+             NCCL_VERSION_CODE >= NCCL_VERSION(2, 28, 0)) {
```

Comment:
Now that I'm thinking about this: This should probably be a runtime check using ncclGetVersion because for XLA-users like JAX NCCL is dynamically linked and the version we compile against is not necessarily the version the is used at runtime. WDYT?